packages/core/test/forEach.spec.ts

```diff
@@ -5,6 +5,8 @@ beforeEach(() => {
   // reset globals
   ;((window as any).DD_LOGS as any) = {}
   ;((window as any).DD_RUM as any) = {}
+  // prevent 'Some of your tests did a full page reload!' issue
+  window.onbeforeunload = () => 'stop'
```

Comment:
Do you have any hint why the navigation occurs?

Reply:
Nope, it was systematic on safari with karma@6.0.1 and I am pretty sure I have seen this error from time to time on ci failure before.
I guess It should not hurt to keep it.